deploy/helm/sumologic/conf/logs/otelcol/config.yaml

```diff
@@ -205,6 +205,14 @@ processors:
     container_annotations:
       enabled: {{ .Values.sumologic.logs.container.perContainerAnnotationsEnabled }}
       prefixes: {{ toJson .Values.sumologic.logs.container.perContainerAnnotationPrefixes }}
+  ## logstransformprocessor is experimental, but the same functionality in transformprocessor doesn't offer robust enough error handling
+  ## TODO: use transformprocessor here after https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16519 is closed
+  logstransform/containers_parse_json:
+    operators:
+      - type: json_parser
+        parse_from: body
+        parse_to: body
+        if: body matches "^{.*"
```

Comment:
Does it produce any logs in case of parsing error? How many?

Reply:
It generates a strack trace.

Reply:
Hmm, log collector's logs are not json so this is going to be logs generator at some point

Reply:
but this is in metadata logs, so we just need to ensure we filter it out before, especially because we don't treat it as error

Reply:
I don't understand this point, this only applies to logs which start with a curly brace.

Reply:
oh, that's right 🤦

Reply:
LGTM then